src/Group.php

```diff
@@ -62,6 +62,11 @@ public static function getTypeName($nb = 0)
         return _n('Group', 'Groups', $nb);
     }
 
+    public static function getSectorizedDetails(): array
+    {
+        return ["admin", "group"];
+    }
```

Comment:
Apparently I can even do better: centralize everything in the existing listener and widening it to CommonDBTM instead of just CommonDropdown, and just add an "if" statement to distinguish between DropdownController and GenericListController.

Later on, in another PR, we might "merge" dropdown and generic list controllers

Reply:
`DropdownController` does the exact same as `GenericListController`, except that it displays the header using a different method.